### PR TITLE
{monitor}`az monitor log-profiles`: Migrate from codegen V2

### PR DESCRIPTION
**Pull request by AllyW**
**Related command**

az monitor log-profiles create/list/show/delete

**Description**

Migrate log-profiles from codegen v2

**Testing Guide**

**History Notes**

[monitor] `az monitor log-profiles`: Migrate create, show, delete, list cmds

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
monitor

File: src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "monitor log-profiles list",
+)
+class List(AAZCommand):
+    """List the log profiles. :keyword callable cls: A custom type or function that will be passed the direct response:return: An iterator like instance of either LogProfileResource or the result of cls(response) :rtype:  ~azure.core.paging.ItemPaged[~$(python-base-namespace).v2016_03_01.models.LogProfileResource] :raises ~azure.core.exceptions.HttpResponseError:.
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py:18]: It seems the command help has invalid content.

File: src/azure-cli/azure/cli/command_modules/monitor/commands.py [hunk 2]
```diff
@@ -244,10 +244,10 @@ def load_command_table(self, _):
...
     with self.command_group('monitor log-profiles', log_profiles_sdk, custom_command_type=log_profiles_custom) as g:
-        g.custom_command('create', 'create_log_profile_operations')
-        g.command('delete', 'delete')
-        g.show_command('show', 'get')
-        g.command('list', 'list')
+        # g.custom_command('create', 'create_log_profile_operations')
+        # g.command('delete', 'delete')
+        # g.show_command('show', 'get')
+        # g.command('list', 'list')
         g.generic_update_command('update')
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/monitor/commands.py:251]: Why still using this update command?

**Comment by kairu-ms** [comment]
Don't forget to clean up the code in help.py

File: src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py (hunk 1 above)
Comment (AllyW) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py:18]: origin list help message

Comment (AllyW) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py:18]: origin list help message

Comment (AllyW) [reply src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py:18]: origin list help message

**Review by kairu-ms — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**